journalpump/journalpump.py

```diff
@@ -330,6 +334,15 @@ def get_state(self):
             "total_bytes": self.read_bytes,
         }
 
+    def inc_secret_filter_metrics(self):
```

Comment:
Nit: `inc` is a bit ambiguous here, I think `report` or something like that would be clearer.